pkg/internal/goexec/structmembers.go

```diff
@@ -128,12 +128,24 @@ var structMembers = map[string]structInfo{
 			"nextID": "http2_client_next_id_pos",
 		},
 	},
+	"golang.org/x/net/http2/hpack.Encoder": {
```

Comment:
I noticed that sometimes hpack.Encode doesn't have to be the vendor version in Go. Added it just to be on the safe side.